pytest_container/runtime.py

```diff
@@ -280,31 +259,30 @@ def get_image_size(
             else str(image_or_id_or_container)
         )
         return float(
-            check_output(
-                [
-                    self.runner_binary,
-                    "inspect",
-                    "-f",
-                    '"{{ .Size }}"',
-                    id_to_inspect,
-                ]
-            )
-            .decode()
-            .strip()
-            .replace('"', "")
+            self.run_command(
+                "inspect",
+                "-f",
+                '"{{ .Size }}"',
+                id_to_inspect,
+            ).replace('"', "")
         )
 
-    def _get_container_inspect(self, container_id: str) -> Any:
+    def _run_inspect(self, container_id: str) -> Union[Dict[str, Any], None]:
```

Comment:
Technically the return type should be `Optional[Dict[str, Any]]`, but I really don't like the option of us returning `None` here. We should either return an inspect or error out. Otherwise we just push error handling down on the API user.

Reply:
If I throw an error on missing, I need to catch it in prepare_container (and then check if the error is because missing or parse error) or make a second function for the same thing (like it is now). Didn't want to refactor prepare_container() too much.